lib/services/pcmk-dbus.h

```diff
@@ -14,19 +14,32 @@
 #    define DBUS_TIMEOUT_USE_DEFAULT -1
 #  endif
 
+G_GNUC_INTERNAL
```

Comment:
I left it out as a business of `glib` to do the right thing.  Apparently
there are prerequisites that it can use (gcc reveals itself to the code
via some extra macros).  And indeed that's the case (it would be sad if
not, it would mean that pacemaker code was already broken before).